src/components/Profile/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+//Import Material-ui
+import { Box } from "@material-ui/core";
+//Import Component
+import CarrouselProfile from "components/CarrouselProfile";
+//Import Style
+import "./style.scss";
+//Import Data
+import DATACARROUSEL from "lib/constants";
+
+const Profile = () => {
+  const DATA = DATACARROUSEL;
+
+  return (
+    <Box className="Profile" component="section">
```

Comment:
Las clases de CSS van en `kebab-case`, todo en minúscula y separado por guiones

```suggestion
    <Box className="profile" component="section">
```

Reply:
Done!